controllers/topics.controllers.js

```diff
@@ -1,7 +1,13 @@
-const { fetchTopics } = require('../models/topics.models');
+const { fetchTopics, fetchArticles } = require('../models/topics.models');
 
-exports.getTopics = (req, res, next) => {
+exports.getTopics = (req, res) => {
   fetchTopics().then((topics) => {
     res.status(200).send({ topics });
   });
 };
+
+exports.getArticles = (req, res) => {
```

Comment:
Think about splitting this out into different controllers - you have topics.controllers but have included getArticles - maybe make a articles.controllers